index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var fs = require('fs');
+var through2 = require('through2');
+var gettextParser = require('gettext-parser');
+var Handlebars = require('handlebars');
+var jsesc = require('jsesc');
+
+var replaceText = function(chunk, enc, callback) {
+  var template = Handlebars.compile(chunk.toString());
+  var chunkString = '';
+
+  chunkString += 'module.exports = "';
+  chunkString += jsesc(template(), {quotes: 'double'});
```

Comment:
unfamiliar with `jsesc`, the docs say:

> generating the shortest possible valid ASCII-only output

ASCII-only seems like any msgids with a non-ascii char will not get the right translation.